VIM_COMPARISON.org

```diff
@@ -0,0 +1,99 @@
+#+title: Vim Comparison
+
+* Intro
+
+meow is not very similar to [[https://www.vim.org/][Vim]] even though both are modal editing modes. One could compare meow to more to [[https://kakoune.org/][kakoune]].
+
+The main differences to Vim are that there is no dedicated visual mode in meow, but it's included into normal mode. This means almost every movement creates a visual selection on which can act on.
+
+The other main difference is that the language you use in Vim is inversed. When saying something like 4w this would move 4 words forward. In meow this is the other way around you would first give the noun w which already selects the next word and afterwards you give 3 to move 3 more words, which gives a total of 4 words selected. You can repeat this indefinitely. For example if you say 2 and 6 afterwards you select another 8 words forward. Afterwards you decide for an action (or verb) what you want to do on this selection. For example w3c would select 4 words, delete them and put you into insert mode. I Vim however you would tell it the verb first, then the amount of nouns and finally the noun itself. In our example with meow's w3c the Vim equivalent would be c3w.
+In comparison meow has the clear advantage when creating the selection, you already get a visual feedback and then can decide which verb to execute. Or you abort if you selected something wrong. In Vim on the other hand you commit on the verb right away, then you need to count blindly the words (which is harder the more things you want to act on) and finally do the action.
+
+However since many people who want to give meow a try, come from a Vim / Evil background, we want to give an overview of typical Vim keybindings and how they can be achieved in meow.
+
+* Movement
```

Comment:
Would it be possible to turn the following key comparison into an org table? 

At first look, I thought the leftmost column of keys also work in meow. So it could be a bit misleading. 

Something like the below might be better:
| Vim key | Meow Command |
| --------| ------------------|
| `h`        | `meow-left`|

What do you think?

Reply:
I created multiple tables now, one for each headline. It's much more readable now indeed 👍